### DISCUSSION TITLE
two instances of Snipe-IT

### DISCUSSION THREAD

**Discussion by rusheedroberts** [snipe/snipe-it#10487] [Q&A] [answered]
Can I install two instances of Snipe-IT on the same server?

**Comment by uberbrady** [comment, marked as answer]
Yes, that's very possible. Most of the trouble with that is going to be at the webserver-configuration level. You'd probably want two separate databases, but you can put them both on the same database _server_ - e.g. `CREATE DATABASE snipe1....CREATE DATABASE snipe2....`. Snipe-IT won't care about most of that stuff, but getting the webserver to point the appropriate URL's to the appropriate places will require some fiddilng.

**Reply by rusheedroberts** [reply]
thank you so much

**Comment by yosiasz** [comment]
> Can I install two instances of Snipe-IT on the same server?

you can install as many snipeit servers as you want to manage

**Reply by rusheedroberts** [reply]
thank you so much
